src/resources/Conversion/SoleProprietorshipResource.ts

```diff
@@ -22,8 +21,15 @@ export const SoleProprietorshipResource: ResourceIF = {
     orgPersonInfo: {
       orgPersonLabel: 'Proprietor',
       orgTypesLabel: 'Business or Corporation',
-      subtitle: null,
-      helpSection: null
+      subtitle: 'You can change the legal name, mailing and delivery addresses and the email address of the ' +
+        'proprietor. To change to a different proprietor, you must form a new business with that proprietor ' +
+        'and dissolve this registration.',
```

Comment:
I'm waiting to hear back from Roy/Yui on whether they want this blurb here (similar to GP), and whether it should refer to "registration".